deps/zlib/zlib.gyp

```diff
@@ -18,14 +18,15 @@
           'conditions': [
             ['target_arch in "ia32 x64" and OS!="ios"', {
               'defines': [ 'ADLER32_SIMD_SSSE3' ],
+              'cflags': [ '-mssse3' ],
```

Comment:
@anonrig Would you happen to know why these variables (cflags, cflags_cpp) don't take hold here even though we are building under Windows with x64.

Reply:
I think it's a GYP issue. It was never meant to support Clang on Windows AFAIK and cflags are passed differently for it. See https://github.com/targos/node/commit/c1d1878545c9ae432ad78728a42a61fc73acd92e